### PR TITLE
checkstyle: check for star imports #228

### LINKED ISSUE #228: checkstyle: check for star imports
The Java coding standard prohibits star imports. Our checkstyle configuration should check for that.

### PR DESCRIPTION
**Pull request by chao1995**
fixes #228 

Ready for review.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/Parser.java [hunk 1]
```diff
@@ -118,112 +122,6 @@ private Command prepareAdd(String args) {
...
-    private UniqueTagList parseTags(Collection<String> tags) throws IllegalValueException {
-        assert tags != null;
-        final Set<Tag> tagSet = new HashSet<>();
-        for (String tagName : tags) {
-            tagSet.add(new Tag(tagName));
-        }
-        return new UniqueTagList(tagSet);
-    }
-
-    /**
```
Comment (damithc) [inline src/main/java/seedu/address/logic/parser/Parser.java]: Why is this code block deleted? May be your fork is out of sync?

Comment (chao1995) [inline src/main/java/seedu/address/logic/parser/Parser.java]: Yes, my fork is out of sync, so I did a pull, but I guess I made some mistakes when fixing merge conflicts. Should be fixed now.

**Comment by damithc** [comment]
@m133225 any input on this PR?

**Comment by damithc** [comment]
@chao1995 I added a commit message body. In future, remember to add a commit message body explaining the rationale for the change.

**Pull request merged**